### PR TITLE
chore: Update Plugin Name and README.md

### PR DESCRIPTION
**Pull request by mohitsuman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# IntelliJ Knative
+# IntelliJ plugin for Knative & Serverless Functons
```
Comment (lstocchi) [inline README.md:1]:
```suggestion
# IntelliJ plugin for Knative & Serverless Functions
```

**Review by lstocchi — APPROVED** [review]
LGTM, just a typo to be fixed

**Comment by mohitsuman** [comment]
> LGTM, just a typo to be fixed Maybe we should think about changing the plugin name in the marketplace as well.

@lstocchi We should change it. `Knative & Serverless Functions` can be the name. We can remove `by Red Hat`, as the publisher name is already mentioned.

**Comment by lstocchi** [comment]
@mohitsuman Talked with @jeffmaury about renaming. He said that the `by Red Hat` was requested by the marketing/legal team so we should keep it. Is it ok for you to have `Knative & Serverless Functions by Red Hat`?

**Comment by mohitsuman** [comment]
yes if Legal has approved it, we should go ahead with that.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by mohitsuman** [comment]
@lstocchi looks good. we can merge this.

**Pull request merged**